src/app/components/reportesDiarios/encabezadoReportesDiarios.component.ts

```diff
@@ -0,0 +1,312 @@
+        // debugger;
+
+        switch (tipo) {
+            case 'organizacion': {
+                if (value.value !== null) {
+                    this.parametros['organizacion'] = this.organizacion.id;
+                    this.parametros['organizacionNombre'] = this.organizacion.nombre;
+                } else {
+                    this.parametros['organizacion'] = '';
+                    this.parametros['organizacionNombre'] = '';
+                }
+            } break;
+
+            case 'tipoReportes': {
+                if (value.value !== null) {
+                    this.parametros['tipoReportes'] = this.tipoReportes.nombre;
+                } else {
+                    this.parametros['tipoReportes'] = '';
+                }
+            } break;
+
+            case 'mes': {
+                if (value.value !== null) {
+                    this.parametros['mes'] = this.mes.id;
+                    this.parametros['mesNombre'] = this.mes.nombre;
+                } else {
+                    this.parametros['mes'] = '';
+                    this.parametros['mesNombre'] = '';
+                }
+            } break;
+
+            case 'anio': {
+                if (value.value !== null) {
+                    this.parametros['anio'] = this.anio.nombre;
+                } else {
+                    this.parametros['anio'] = '';
+                }
+            } break;
+
+            case 'prestacion': {
+                if (value.value !== null) {
+                    this.parametros['prestacion'] = this.prestacion.id;
+                    this.parametros['prestacionNombre'] = this.prestacion.nombre;
+                } else {
+                    this.parametros['prestacion'] = '';
+                    this.parametros['prestacionNombre'] = '';
+                }
+            } break;
+
+            case 'fecha': {
+                if (value.value !== null) {
+                    this.parametros['fecha'] = this.fecha;
+                } else {
+                    this.parametros['fecha'] = '';
+                }
+            } break;
+
+            default:
+                break;
+        }// End Switch
+
+    }
+
+
+    onChangeTipoReportes(event) {
+
+        this.parametros = {};
+        this.parametros['organizacion'] = this.organizacion.id;
+        this.parametros['organizacionNombre'] = this.organizacion.nombre;
+        if (this.tipoReportes !== null) {
+            this.parametros['tipoReportes'] = this.tipoReportes.nombre;
+        }
+
+    }
+
+
+
+
+
+
+    public generar() {
+
+
+        if (this.parametros['organizacion'] && this.parametros['tipoReportes'] && this.parametros['mes'] && this.parametros['anio'] && this.parametros['tipoReportes'] === 'ResumenDiarioMensual') {
+
+            this.agendaService.findResumenDiarioMensual(this.parametros).subscribe((reporte) => {
+
+                this.reporte = reporte;
+                this.showResumenDiarioMensual = true;
```

Comment:
```suggestion
                this.showPlanillaC1 = false; 
                this.showResumenDiarioMensual = true;
```
Si generan reporte C1 primero y después mensual, se estarían mostrando los dos juntos.